docs/source/guide/active_learning.md

```diff
@@ -76,7 +77,7 @@ In order to maximize the training efficiency and effectiveness of your machine l
 On the project data manager, select **Label All Tasks** to start labeling.
 
 As your model retrains and a new version is updated in Label Studio, the tasks shown next to annotators are always those with the lowest prediction scores, reflecting those with the lowest model certainty. The predictions for the tasks correspond to the latest model version.
-
+</div>
```

Comment:
Read and insert <div class="enterprise-only">or <div class="opensource-only"> .
Important note: before and after div should be newlines, like on the screenshot!

Reference: https://heartex.slack.com/archives/C01HGFXA7KR/p1670507767449669

Reply:
Fixed